deployments/jwebbinar/image/environments/jwebbinar/jdaviz.pip

```diff
@@ -98,6 +99,17 @@ webencodings
 widgetsnbextension
 xlrd
 
+bqplot>=0.12.33
```

Comment:
Is there any reason not to just do these in common-env/jupyter.pip?   The idea there was to capture some of this difficult debug/setup for all the missions.   OTOH,  if that doesn't work,  maybe we should ditch common-env/jupyter.pip?  (Obviously you can do this though if it feels right,  pip-tools will let you have both,  and there are ways to omit jupyter.pip too.)

Reply:
Good point.  I think I had just copied that from the jdaviz environment dir.  I'll give it a shot...

Reply:
This did work.